dev/com.ibm.ws.concurrent.persistent_fat_multiple/test-applications/persistmultitest/src/web/DBUpdateTask.java

```diff
@@ -21,9 +21,13 @@
  * Task that update an entry in the database when it runs.
  */
 public class DBUpdateTask implements Runnable {
+	
+    //Get resource reference of either embedded or networked database for lookup.
+    private static final String resourceRef = System.getenv("DB_REF");
```

Comment:
lets try and get this mechanism working without any application changes necessary. I think the key here is that we use 1 datasource for everything, and not have to swap to a different datasource for Derby vs. other DBs